sdk/cargo-build-bpf/tests/crates.rs

```diff
@@ -0,0 +1,43 @@
+use std::{
+    env, fs,
+    process::{Command, Output},
+};
+
```

Comment:
It would be nice to remove the forward slash path separator all through these tests, to give the person who finally decides to take on adding Windows support less work.  But I don't feel too strong about this if you want to leave it as is for now

Reply:
I'll fix this.